VirtualEconomyFramework/VENFTApp-Blazor/Components/Align.cs

```diff
@@ -0,0 +1,7 @@
+﻿public enum Align
+{
+    Default,
+    Left,
+    Right,
+    Center,
+}
```

Comment:
This enum is used to define Column Alignment in Table

Reply:
ok. maybe we should place all these help class in the specific folder in the Components folder. just idea...